.github/workflows/tests.yaml

```diff
@@ -49,7 +49,7 @@ jobs:
         runs-on: ubuntu-latest
 
         container:
-            image: ghcr.io/project-chip/chip-build:54
+            image: ghcr.io/project-chip/chip-build:74
```

Comment:
This one sweems to switch from ubuntu 20.04 to 24.04 and I seem to get an error in a python test case (which makes no sense, however I am not familiar with it). I am re-running it the 3rd time now, however it failed 2 times in a row, so it may somehow be a legitimate error, potentially related to some python functionality likely (just a guess .. no idea what is going on, but python being different seems more likely than apps being different).

We may want to create an issue for this and not update it yet.

Reply:
TestPurposefulFailureExtraReportingOnToggle seems to be the unexpected passing test in CI.

Reply:
I can reproduce this issue on my local machine as well (which is a recent ubuntu-based).
I believe this test case may be in error however I am not familiar with it.

Reply:
test added as part of #31162 by @bzbarsky-apple 
it seems the intent is that we should not report attribute changes when the attribute does not change.

Reply:
Debugging locally, it seems like the parser stops completely at `Running Check for attribute report` (it does not seem to actually be able to pass that check at all, but just stops without an exception). I am unclear why.

Reply:
The test seems to timeout (get no instance.recv in  WebSocketRunner::execute), yet no timeout exception is thrown.

Reply:
what I observe is that https://github.com/project-chip/connectedhomeip/blob/master/scripts/py_matter_yamltests/matter_yamltests/runner.py#L225 finishes after 5 seconds (which is what the test calls for) and no exception is raised.

Reply:
Do I need to revert it from 74 back to 54 for now?

Reply:
I think #35045 contains a fix for this.